README.md

```diff
@@ -316,8 +320,58 @@ The Realm Dart package is `realm_dart`
 
     _*This file should be committed to source control_
 
-* For more usage of Realm Dart see the Realm Flutter usage above.
+* For more information on using the Realm Dart Standalone SDK see the Realm Flutter SDK usage above.
+
+# Device Sync with Realm Flutter and Dart Standalone SDKs
+
+Run the Realm with [Device Sync](https://www.mongodb.com/docs/realm/sdk/flutter/sync/) and connecting to [Atlas App Services](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/).
+
+## Set up Atlas App Services
+
+* Create an account on [cloud.mongodb.com](https://cloud.mongodb.com) - follow the instructions: [Get Started with Atlas](https://www.mongodb.com/docs/atlas/getting-started) or [Get Started with a template app](https://www.mongodb.com/docs/atlas/app-services/tutorial/flutter/)
+1. Create a new App following the instructions here: [Create an App with Atlas App Services UI](https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-realm-ui) or [Create an App with a Template App](https://www.mongodb.com/docs/atlas/app-services/tutorial/flutter/).
+* Read [Authentication Providers](https://www.mongodb.com/docs/atlas/app-services/authentication/providers/) to see how to configure the appropriate authentication provider type.
+* Go to the `Device Sync` menu and [Enable Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/#enable-flexible-sync).
+* [Find and Copy the App ID](https://www.mongodb.com/docs/atlas/app-services/reference/find-your-project-or-app-id/) of your new application.
+
+## Use Device Sync in the Realm SDK
+
+Set Atlas App Id as a constant
+
+``` dart
+  String appId = "tasktracker-fleld";
```

Comment:
I suggest we provide longer example here. so combining this example with the next one will be more clear.

Reply:
Moved to the next part of code.